.circleci/config.yml

```diff
@@ -89,47 +89,51 @@ installdeps: &installdeps
       python setup.py develop
       python -c "import nltk; nltk.download('punkt')"
 
-installtorchgpu14: &installtorchgpu14
+installtorchgpu16: &installtorchgpu16
   run:
     name: Install torch GPU and dependencies
     command: |
-      python -m pip install --progress-bar off 'torch==1.4.0'
-      python -m pip install --progress-bar off 'subword-nmt==0.3.7'
+      python -m pip install --progress-bar off torch==1.6.0+cu101 torchvision==0.7.0+cu101 -f https://download.pytorch.org/whl/torch_stable.html
+      python -m pip install --progress-bar off 'torchtext==0.7.0'
       python -m pip install --progress-bar off pytorch-pretrained-bert
-      python -m pip install --progress-bar off 'torchtext==0.5.0'
-      python -m pip install --progress-bar off 'torchvision==0.5.0'
       python -m pip install --progress-bar off transformers
       python -m pip install --progress-bar off fairseq
       python -c 'import torch; print("Torch version:", torch.__version__)'
       python -m torch.utils.collect_env
 
 
-installtorchgpu13: &installtorchgpu13
+installtorchgpu15: &installtorchgpu15
   run:
     name: Install torch GPU and dependencies
     command: |
-      python -m pip install --progress-bar off 'torch==1.3.1'
-      python -m pip install --progress-bar off 'subword-nmt==0.3.7'
-      python -m pip install --progress-bar off pytorch-pretrained-bert
-      python -m pip install --progress-bar off transformers
-      python -m pip install --progress-bar off fairseq
+      python -m pip install --progress-bar off 'torch==1.5.1'
+      python -c 'import torch; print("Torch version:", torch.__version__)'
+      python -m torch.utils.collect_env
+
+installtorchgpu14: &installtorchgpu14
```

Comment:
Hmm we still want to support 1.4 instead of just supporting the last 2 versions?

Reply:
I don't want to force the entire team to upgrade right now.